### PR TITLE
Display detail attachments

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      selectedTabId={category || list[0].name}
+      className="attachment"
+    >
+      {list.map(({ name, attachments, id: categoryId }) => (
+        <Tab
+          key={categoryId}
+          id={name}
+          title={<NavLink to={generateURI('layer', { layer, id, action, section, category: name })}>{name}</NavLink>}
+          panel={(
+            <ul className="attachment__list">
```
Comment (mabhub) [inline src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js]: This should [definitely](https://giphy.com/gifs/random-french-music-pierre-billon-11sKNq5kL1r904) be a dedicated component.

File: src/modules/CRUD/components/Details/Read/Read.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
-import React from 'react';
+import React  from 'react';
```
Comment (mabhub) [inline src/modules/CRUD/components/Details/Read/Read.js]: wut?

File: src/modules/CRUD/components/Details/Read/Read.js [hunk 3]
```diff
@@ -25,7 +27,15 @@ const Read = ({
   return (
     <div className="details">
       <Header title={title} documents={documents} />
-      <DefaultView properties={properties} />
+      <Menu section={section} />
+      <div className="details__content">
+        {section === 'default'
+          ? <DefaultView properties={properties} />
+          : (
```
Comment (mabhub) [inline src/modules/CRUD/components/Details/Read/Read.js]:
```js
      <div className="details__content">
        {section === 'default'
          ? <DefaultView properties={properties} />
          : <AttachmentView />
        }
      </div> 
```

File: src/modules/CRUD/components/Details/Menu/Menu.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+];
+
+const Menu = ({
+  section,
+  match: { params: { layer, id, action = 'read' } },
+  t,
+}) => (
+  <Navbar className="details__menu">
+    <NavbarGroup>
+      {nav.map(({ text, icon, section: itemSection }) => (
```
[the remark this answers is not in the document]
Comment (dtrucs) [reply src/modules/CRUD/components/Details/Menu/Menu.js]: I'm not sure to apply this modification. There are all props to spread, it doesn't really help to read the code with this split imho.

File: src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+  useEffect(() => {
+    if (categories) {
+      return;
+    }
+    const getCategories = async () => {
+      const result = await getAttachmentCategories(attachmentCategoriesEndpoint);
+      setCategories(result);
+    };
+    getCategories();
```
[the remark this answers is not in the document]
Comment (dtrucs) [reply src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js]: You are suggesting to end the function with a return like the following?
`return () => { didCancel = true; };`

File: src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js (hunk 1 above)
Comment (dtrucs) [reply src/modules/CRUD/components/Details/Read/AttachmentView/AttachmentView.js]: Yep, but waiting for an UX/UI review, this component will [definitely](https://giphy.com/gifs/random-french-music-pierre-billon-11sKNq5kL1r904) be changed in the future :-)

File: src/modules/CRUD/components/Details/Menu/Menu.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (mabhub) [reply src/modules/CRUD/components/Details/Menu/Menu.js]: Separating code behaviours (nav.map) from component rendering always seems a readability improvement for me.

It's also opening the door for easier component re-use.

_Anyway, it is a non-blocking remark from me._

**Pull request merged**